templates/lxc-download.in

```diff
@@ -224,11 +231,9 @@ EOF
     return 0
 }
 
-options=$(getopt -o d:r:a:hl -l dist:,release:,arch:,help,list,variant:,\
+if ! options=$(getopt -o d:r:a:hl -l dist:,release:,arch:,help,list,variant:,\
```

Comment:
I like it when concise, and because I thought the contrary at first. Defining the assignation like this allowed me to better understand what it did. At first, I thought this check was related to the content of `options` but actually it is related to the return code of `getopt`.
Also, shellcheck advises to use this syntax instead.
```
if [ $? -ne 0 ]; then
     ^-- SC2181: Check exit code directly with e.g. 'if mycmd;', not indirectly with $?.
```